API_v2.md

```diff
@@ -1171,10 +1203,66 @@ curl -s localhost:9090/wallet \
 {
   "method": "submit_transaction",
   "result": {
-    "transaction": {
-      "transaction_log_id": "96df759d272cfc134b71e24374a7b5125fe535f1d00fc44c1f12a91c1f951122"
+    "transaction_log": {
+      "object": "transaction_log",
+      "transaction_log_id": "ab447d73553309ccaf60aedc1eaa67b47f65bee504872e4358682d76df486a87",
+      "direction": "tx_direction_sent",
+      "is_sent_recovered": null,
+      "account_id": "a8c9c7acb96cf4ad9154eec9384c09f2c75a340b441924847fe5f60a41805bde",
+      "recipient_address_id": "CaE5bdbQxLG2BqAYAz84mhND79iBSs13ycQqN8oZKZtHdr6KNr1DzoX93c6LQWYHEi5b7YLiJXcTRzqhDFB563Kr1uxD6iwERFbw7KLWA6",
+      "assigned_address_id": null,
+      "value_pmob": "42000000000000",
+      "fee_pmob": "10000000000",
+      "submitted_block_height": "152950",
+      "finalized_block_height": null,
```

Comment:
Per our conversation, this should be submitted_block_index and finalized_block_index in an upcoming PR.